faqs/how-does-eth-2-reduce-eneregy-consumption.md

```diff
@@ -3,6 +3,7 @@ title: How does 2.0 reduce the energy consumption of the Ethereum blockchain?
 weight: 6.0
 attribution:
   -
-    name:
-    link:
----
\ No newline at end of file
+    name: Lamboshi
+    link: https://twitter.com/L_Nakaghini
+---
+In Proof of work mining, whoever solves the block first gets the reward. Simply put, it is a race and if you have more hash rate than someone else you are more likely to win. The end result of the race mechanism is that miners are running as many GPUs as they can get at 100% load, 24 hours a day and the power usage will continue to scale with the price of the block rewards they earn. In proof of stake, the block proposers are randomly selected which removes the race condition. There is no way to increase the likelihood that you get chosen to produce a block, so there is no need to consume more and more energy to improve your chances.
```

Comment:
- an educational source for "the power usage will continue to scale with the price of the block rewards they earn" because that's a _great_ point.
- ditto for the random selection aspect of PoS